--- koji-1.12.0/cli/koji.conf.orig	2017-04-18 14:35:15.879593521 -0500
+++ koji-1.12.0/cli/koji.conf	2017-04-18 14:37:45.818233378 -0500
@@ -3,18 +3,20 @@
 ;configuration for koji cli tool
 
 ;url of XMLRPC server
-;server = http://hub.example.com/kojihub
+server = https://koji.fedoraproject.org/kojihub
 
 ;url of web interface
-;weburl = http://www.example.com/koji
+weburl = https://koji.fedoraproject.org/koji
 
 ;url of package download site
-;pkgurl = http://www.example.com/packages
+topurl = https://kojipkgs.fedoraproject.org/
 
 ;path to the koji top directory
 ;topdir = /mnt/koji
 
 ;configuration for Kerberos authentication
+authtype = kerberos
+krb_rdns = false
 
 ;the service name of the principal being used by the hub
 ;krbservice = host
@@ -25,10 +27,4 @@
 ;the keytab to auth as for automated clients
 ;keytab = /etc/krb5.keytab

-;configuration for SSL authentication
-
-;client certificate
-;cert = ~/.koji/client.crt
-
-;certificate of the CA that issued the HTTP server certificate
-;serverca = ~/.koji/serverca.crt
+use_fast_upload = yes
